variables.tf

```
@@ -72,3 +72,9 @@ variable "sku" {
   default = "Standard"
   description = "SKU type(Basic, Standard and Premium)"
 }
+
+variable "mgmt_subscription_id" {}
+
+variable "aks_infra_subscription_id" {}
```

the jenkins library will set this for you no need to set in each environment

🚫 Blocker
```suggestion
variable "aks_subscription_id" {}
```